elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1594,13 +1597,22 @@ private boolean isValidParameterizedMap(Map<?, ?> values, Class<?> keyType, Clas
      * Invoke the get[fieldName] method on the target object OR get the field with the corresponding name.
      * @param target the object to get
      * @param fieldName the field name to get or invoke equivalent get method
-     * @param dictionary the dictionary
+     * @param requestScope the request scope
      * @return the value
      */
-    public static Object getValue(Object target, String fieldName, EntityDictionary dictionary) {
+    public static Object getValue(Object target, String fieldName, RequestScope requestScope) {
+        EntityDictionary dictionary = requestScope.getDictionary();
         AccessibleObject accessor = dictionary.getAccessibleObject(target, fieldName);
         try {
             if (accessor instanceof Method) {
+                // Pass RequestScope into @Computed fields if requested
+                if (Stream.of(accessor.getAnnotations())
+                        .map(Annotation::annotationType)
+                        .anyMatch(c -> c == ComputedAttribute.class || c == ComputedRelationship.class)
```

Comment:
We do this in `EntityBinding` too, do we want to save the value there so we can retrieve it more efficiently? Perhaps also remembering if the computed thing takes the `RequestScope` too?